tests/fgcmcalTestBase.py

```diff
@@ -422,17 +422,20 @@ def _testFgcmOutputProducts(self, visitDataRefName, ccdDataRefName, filterMappin
         # airmass.  Furthermore, this is a very rough comparison because
         # the look-up table is computed with very coarse sampling for faster
         # testing.
-        # Therefore, this rough comparison can only be seen as a sanity check
-        # and is not high precision.
-        self.assertFloatsAlmostEqual(testResp, lutCat[0]['atmStdTrans'], atol=0.06)
+
+        # To account for overall throughput changes, we scale by the median ratio,
+        # we only care about the shape
+        ratio = np.median(testResp/lutCat[0]['atmStdTrans'])
+        self.assertFloatsAlmostEqual(testResp/ratio, lutCat[0]['atmStdTrans'], atol=0.04)
 
         # The second should be close to the first, but there is the airmass
         # difference so they aren't identical
         testTrans2 = butler.get('transmission_atmosphere_fgcm',
                                 dataId={visitDataRefName: visitCatalog[1]['visit']})
         testResp2 = testTrans2.sampleAt(position=geom.Point2D(0, 0),
                                         wavelengths=lutCat[0]['atmLambda'])
-        self.assertFloatsAlmostEqual(testResp, testResp2, atol=1e-4)
+        ratio = np.median(testResp/testResp2)
+        self.assertFloatsAlmostEqual(testResp/ratio, testResp2, atol=0.04)
```

Comment:
I'm not sure I follow this change. The tolerance got much looser, and now you're testing a ratio? I think it might want a comment.